src/Compilers/Core/Portable/PEWriter/MetadataWriter.cs

```diff
@@ -2336,7 +2320,8 @@ private void PopulateFieldMarshalTableRows()
             }
         }
 
-        private void PopulateFieldRvaTableRows(BlobBuilder mappedFieldDataWriter)
+#nullable enable
+        private void PopulateFieldRvaTableRows(ref PooledBlobBuilder? mappedFieldDataWriter)
```

Comment:
>mappedFieldDataWriter [](http://example.com/codeflow?start=70&length=21)

Do we ever call this method with non-null value?

Reply:
Good catch. No we don't. Can make this clearer with `out`.